zend_abstract_interface/zai_compat.h

```diff
@@ -1,17 +1,20 @@
 #ifndef ZAI_COMPAT_H
 #define ZAI_COMPAT_H
 
-// macros to be used only within ZAI headers or tests to avoid duplication of code lines
```

Comment:
Any callsite would need to use TSRMLS_ junk at the callsite, so it's not true that these must only be used in tests.